cumulus/parachains/runtimes/starters/seedling/src/lib.rs

```diff
@@ -216,7 +216,10 @@ impl pallet_aura::Config for Runtime {
 impl pallet_timestamp::Config for Runtime {
 	type Moment = u64;
 	type OnTimestampSet = Aura;
+	#[cfg(feature = "experimental")]
 	type MinimumPeriod = ConstU64<0>;
+	#[cfg(not(feature = "experimental"))]
+	type MinimumPeriod = ConstU64<{ RELAY_CHAIN_SLOT_DURATION_MILLIS / 2 }>;
```

Comment:
I think it's better to use `parachains_common::SLOT_DURATION` instead of `RELAY_CHAIN_SLOT_DURATION_MILLIS`.